src/components/Header/Header.js

```diff
@@ -0,0 +1,23 @@
+import styled from 'styled-components/macro'
+
+export default function Header({ title }) {
+  return (
+    <AppHeader>
```

Comment:
If you only have one heading in your header component, you should think about just using the heading element without the header. Usually you have multiple elements in a `header` element such as a subheadline, a navigation and so on. In this case the `header` element is just wrapping one other element and not really necessary. This would then be your `Headline` component instead of `Header`, if that makes sense?